From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 08 Jun 2016 21:04:00 +0200
Subject: [PATCH] Do not install METIS

METIS is optional (and not needed for Julia), but `make install` fails when
it's not built, so we must remove it from Makefile.
--- a/Makefile
+++ b/Makefile
@@ -56,19 +56,6 @@
 #	( cd SKYLINE_SVD && $(MAKE) install )
 	$(CP) README.txt $(INSTALL_DOC)/SuiteSparse_README.txt
 	chmod 644 $(INSTALL_DOC)/SuiteSparse_README.txt
-ifeq (,$(MY_METIS_LIB))
-        # install METIS from SuiteSparse/metis-5.1.0
-	- $(CP) lib/libmetis.* $(INSTALL_LIB)
-	- $(CP) metis-5.1.0/manual/manual.pdf $(INSTALL_DOC)/METIS_manual.pdf
-	- $(CP) metis-5.1.0/README.txt $(INSTALL_DOC)/METIS_README.txt
-        # the following is needed only on the Mac, so *.dylib is hardcoded:
-	$(SO_INSTALL_NAME) $(INSTALL_LIB)/libmetis.dylib $(INSTALL_LIB)/libmetis.dylib
-	- $(CP) include/metis.h $(INSTALL_INCLUDE)
-	chmod 755 $(INSTALL_LIB)/libmetis.*
-	chmod 644 $(INSTALL_INCLUDE)/metis.h
-	chmod 644 $(INSTALL_DOC)/METIS_manual.pdf
-	chmod 644 $(INSTALL_DOC)/METIS_README.txt
-endif
 
 # uninstall all packages
 uninstall:
